app/assets/stylesheets/mobile.scss

```diff
@@ -43,4 +43,5 @@ $success: #00911f;
 // app/assets/stylesheets/notes.scss
 @import 'notes';
 // app/assets/stylesheets/home.scss
-@import 'home';
\ No newline at end of file
+@import 'home';
```

Comment:
not sure if they are necessary, especially the desktop variant?

Reply:
Yes, should be removed as otherwise we have that Stylesheet loaded in all mobile views.